site/docs/styles.css

```diff
@@ -48,13 +48,38 @@
   padding: 8px 8px 0;
 }
 
+.vocs_DesktopTopNav_item.vocs_NavigationMenu_link.vocs_Link.vocs_Link_styleless {
+  color: #030712;
+}
+
+.dark .vocs_DesktopTopNav_item.vocs_NavigationMenu_link.vocs_Link.vocs_Link_styleless {
+  color: #F9FAFB;
+}
+
 .vocs_Tabs_trigger {
   padding: 8px 16px;
   border: none;
   cursor: pointer;
   border-top-left-radius: 4px;
   border-top-right-radius: 4px;
   transition: color 0.3s;
+  color: #030712;
+}
+
+#home-footer-list .vocs_Anchor.vocs_Link.vocs_Link_accent_underlined {
```

Comment:
Sorry, ID CSS are a must no in the CSS community 😄
Always avoid to use them as ID have very odd side effects.

https://medium.com/@clairecodes/reasons-not-to-use-ids-in-css-a42204fb0d97#:~:text=IDs%20should%20be%20unique%20on,of%20the%20advantages%20of%20CSS.

Reply:
oo interesting, ok will remove!